stable/redmine/templates/secrets.yaml

```diff
@@ -15,5 +15,10 @@ data:
   {{ else }}
   redmine-password: {{ randAlphaNum 10 | b64enc | quote }}
   {{ end }}
+  {{ if .Values.externalDatabase.password }}
+  external-db-password: {{ default "" .Values.externalDatabase.password | b64enc | quote }}
+  {{ else }}
+  external-db-password: {{ randAlphaNum 10 | b64enc | quote }}
+  {{ end }}
```

Comment:
Suggestion:
```diff
+ {{- if or (and .Values.databaseType.mariadb (not .Values.mariadb.enabled)) (and .Values.databaseType.postgresql (not .Values.postgresql.enabled)) }}
   {{- if .Values.externalDatabase.password }}
   external-db-password: {{ default "" .Values.externalDatabase.password | b64enc | quote }}
   {{- else }}
   external-db-password: {{ randAlphaNum 10 | b64enc | quote }}
   {{- end }}
+  {{- end }}
```

Reply:
You are correct. Added if statement for "external-db-password".